### PR TITLE
New package: hunspell-ro_RO-3.3.10

### PR DESCRIPTION
**Pull request by klorophatu**

### CODE REVIEW

File: srcpkgs/hunspell-ro_RO/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'hunspell-ro_RO'
+pkgname=hunspell-ro_RO
+version=3.3.10
+revision=1
+wrksrc=hunspell-ro.${version}
+short_desc="Romanian ro_RO dictionary for hunspell"
```
Comment (ericonr) [inline srcpkgs/hunspell-ro_RO/template:6]: Does it need `ro_RO` in the description?

**Review by ericonr — COMMENTED** [review]
Be sure to squash your commits into a single one for the PR!

File: srcpkgs/hunspell-ro_RO/template (hunk 1 above)
Comment (klorophatu) [reply srcpkgs/hunspell-ro_RO/template:6]: Not really, I just saw that some other dictionaries had it and decided to add it. I suppose it's unnecessary.

**Comment by ericonr** [comment]
(please fix the commit message as well) :)

File: srcpkgs/hunspell-ro_RO/template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+maintainer="Klorophatu <klorophatu@krutt.org>"
+license="GPL-2.0-or-later, LGPL-2.1-or-later, MPL-1.1"
+homepage="https://sourceforge.net/projects/rospell/"
+distfiles="https://sourceforge.net/projects/rospell/files/Romanian%20dictionaries/dict-${version}/hunspell-ro.${version}.tar.gz"
```
Comment (ericonr) [inline srcpkgs/hunspell-ro_RO/template:10]: We have `SOURCEFORGE_SITE` as a variable you can use for downloading stuff from sourceforge, do you think it's possible to plug it in here?

**Comment by ericonr** [comment]
CI is having weird issues :/

Also, please ping me when you have made the requested changes :)

**Comment by klorophatu** [comment]
@ericonr 

`$SOURCEFORGE_SITE` is `https://downloads.sourceforge.net/sourceforge`. The subdomain `downloads` doesn't work but `download` does. What should I do?

File: srcpkgs/hunspell-ro_RO/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'hunspell-ro_RO'
+pkgname=hunspell-ro_RO
+version=3.3.10
+revision=1
+create_wrksrc=yes
+hostmakedepends="unzip"
```
Comment (ericonr) [inline srcpkgs/hunspell-ro_RO/template:6]: Why did you add this? We use `bsdtar` for zip files.

And I believe it was working without `create_wrksrc` previously?

File: srcpkgs/hunspell-ro_RO/template [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+maintainer="Klorophatu <klorophatu@krutt.org>"
+license="GPL-2.0-or-later, LGPL-2.1-or-later, MPL-1.1"
+homepage="https://sourceforge.net/projects/rospell/"
+distfiles="${SOURCEFORGE_SITE}/project/rospell/Romanian%20dictionaries/dict-${version}/ro_RO.${version}.zip"
+checksum=7f128d64ea06c9e6711c30b118c0afeefb014d8f33c92daccdf455aba2d04519
+
+do_install() {
+	vinstall ${wrksrc}/ro_RO.aff 644 /usr/share/hunspell
+	vinstall ${wrksrc}/ro_RO.dic 644 /usr/share/hunspell
+	vlicense ${wrksrc}/README
```
Comment (ericonr) [inline srcpkgs/hunspell-ro_RO/template:17]: I'm not sure `${wrksrc}` should be required here...

Also, use `usr/share/hunspell`.

**Review by ericonr — COMMENTED** [review]
If `SOURCEFORGE_SITE` generates too many failures, feel free to use the previous link.

**Comment by klorophatu** [comment]
@ericonr 

Sourceforge keeps timing out, making the builds fail.
